app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,90 @@
+class TweetsController < ApplicationController
+
+  # CREATE
+
+  get "/tweets/new" do # (must validate session ID)
+    @flash_message = session[:flash] if session[:flash]
+    session[:flash] = nil
+    if UserHelper.logged_in?(session)
+      @flash_message = session[:flash] if session[:flash]
+      session[:flash] = nil
+      erb :'/tweets/new'
+    else
+      redirect "/login"
+    end
+  end
+
+  post "/tweets" do
+    @tweet = Tweet.new(params)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = UserHelper.current_user.tweets.build(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.